### ISSUE TITLE
More gesture control options

### ISSUE THREAD

**Issue by ayy91** [Adamantcheese/Kuroba#91]
Add the below (optional) gesture controls:
- hold thread body/text on catalog to pin
- swipe left on a post to quote/unquote
- swipe left and hold for 1 second to quote/unquote text
- shake phone to refresh

**Comment by Adamantcheese** [comment]
1) Will likely be more annoying than anything, as the long-tap image to copy URL feature is an example of. It'll cause a lot of accidental bookmarking, which is WAY more annoying than overwriting your clipboard. Won't implement.
2) Because of the slide mode layout this will likely cause lots of weird issues. Will consider.
3) This is just profoundly counterintuitive to how most touch controls work, which is one touch action for one application event. Chaining them together means that nobody's going to figure it out and then I'll get more requests for even dumber gesture actions, and I don't want a phone application to turn into a fighting game character's move sheet. Won't implement.
4) No. Just tap the refresh button and don't shake your phone like an idiot. Absolutely will not implement.

In summary, the only action that will be considered of the three is swipe left to quote/unquote.

**Comment by ownBard** [comment]
I just want to be able to select the text I want to quote and then click the post number to quote it (like in a normal browser). Kuroba won't even let me select text unless I first open the reply box for whatever reason.

**Comment by Adamantcheese** [comment]
@ownBard This is unrelated to this issue. Submit a new issue.

**Comment by ownBard** [comment]
I meant as an alternative to his third point. If it's unsuitable then it would just make for another unnecessary feature requests.

**Comment by Adamantcheese** [comment]
If you select text in a post cell, the context menu that appears will have a quote option. If you can't select text in normal post cells, that's something wrong on your end, because that feature is exactly the same as it was in Clover. You aren't able to select text from post cells when the reply popup is open, this is known and was judged not a necessary fix for the time being. When you press the quote button, it will automatically quote the post number and the selected text.

**Comment by ownBard** [comment]
@Adamantcheese  So once again I'm caught suggesting something that is already implemented but just doesn't work on my phone. Heh nevermind then...

**Comment by Adamantcheese** [comment]
When I closed #125, I mentioned the tap post number to reply option. This should be sufficient for quoting, and unquoting isn't really something that should happen accidentally in normal use cases. Closing for those reasons.
